src/main/java/uk/gov/ida/eventemitter/SqsClient.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.ida.eventemitter;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+import com.google.inject.Inject;
+
+import javax.inject.Named;
+
+public class SqsClient {
```

Comment:
Question: How are we expecting the AmazonSQS to be provided?
Is that not something that we should be including in our module?


perhaps this is a larger question about how we are going to let clients configure our library!

Reply:
It is best to leave clients to configure SQS. They need to provide queue URL required for Amazon SQS, to choose whether to send or not to send events to a Amazon SQS. They also need to have an ability to choose another class implementing AmazonSQS interface class e.g. to use it for integration testing or real testing.